ts/components/wallet/PaymentMethodsList.tsx

```diff
@@ -16,7 +16,13 @@ import {
 } from "react-native";
 import { SvgProps } from "react-native-svg";
 import { connect } from "react-redux";
-import { Icon, HSpacer, VSpacer, Divider } from "@pagopa/io-app-design-system";
+import {
+  Icon,
+  HSpacer,
+  VSpacer,
+  Divider,
+  H6
```

Comment:
In that case, we have H3 and H5 that became H6, which appears to be an error but we can ignore these changes as this PR #6210 will remove the file.